### DISCUSSION TITLE
DWARF 1.0 / 1.1 Debugging Data Format Support

### DISCUSSION THREAD

**Discussion by travelsonic** [NationalSecurityAgency/ghidra#3512] [Ideas]
I am actually surprised that only DWARF 2.0 upward is supported.  It'd be cool if it were supported rather than having to rely on plugins that only work on older versions of Ghidra (and old versions of Ghidra) to be able to take advantage of DWARF 1.0/1.1 format debugging data in REing programs that have DWARF debugging data, but older 1.0/1.1 format DWARF data, in it.

**Comment by ryanmkurtz** [comment]
Duplicate of #2985
